src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationHandler.cs

```diff
@@ -168,15 +168,15 @@ protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OA
         /// </summary>
         /// <param name="message">The message.</param>
         /// <param name="elementName">Name of the element.</param>
-        /// <returns>Task&lt;JsonDocument&gt;.</returns>
-        private async Task<JsonDocument> ParseJsonAsync(HttpResponseMessage message, string elementName)
+        /// <returns>Task&lt;System.String&gt;.</returns>
```

Comment:
`<returns><see cref="Task{T}"/></returns>`, though it's not adding much to this method here so maybe just remove the comments from it?

Reply:
Removed.